front-end-admin/src/components/dashboard/dashboard-tiles-card.tsx

```
@@ -38,50 +38,7 @@ function DashboardTilesCard() {
-            withdrawalsData?.dailyAverageComparisonPercentageWithdrawals || 0.00
-          )}
-          label="Amount Received "
-          tooltipText={`${formatTiles(
-            withdrawalsData?.dailyAverageComparisonPercentageWithdrawals || 0.00
-          )}% than last month`}
-          tippyBgColor={getBgColor(
-            withdrawalsData?.dailyAverageComparisonPercentageWithdrawals || 0.00
-          )}
-          icon={TileSVG4}
-        />
+        // in progress
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)